### DISCUSSION TITLE
[feat] Support for `net` to create a TCP Socket

### DISCUSSION THREAD

**Discussion by lnbc1QWFyb24** [tauri-apps/tauri#14547] [Plugin Requests]
### Describe the problem

It would be great to make use of native TCP sockets instead of WebSockets.

### Describe the solution you'd like

Is it possible to add a `net` module API?

### Alternatives considered

Maybe it is pretty simple to expose a TCP socket via a Rust invoke command? Maybe there is already a plugin for this? Please point me if there is something already out there, or if I need to go and learn Rust :)

### Additional context

Just a massive thanks for all your great work. The dev experience with Tauri is amazing!

**Comment by amrbashir** [comment]
I don't think this will be available in 1.x release so I will transfer to `plugins-workspace` where it could have its own plugin in v2 maybe

**Comment by maxfriedmann** [comment]
@lnbc1QWFyb24 you should have a look at https://github.com/kuyoonjo/tauri-plugin-tcp, did the trick for me!
